fw/http_match.c

```diff
@@ -952,7 +952,7 @@ do {									\
 			if (tfw_str_eq_cstr(&tmp, cstr, clen,
 					    TFW_STR_EQ_PREFIX))
 			{
-				TFW_COLLECT_COOKIE(req, chunk, val, n);
+				TFW_COLLECT_REQ_COOKIE(req);
```

Comment:
I know it's intentional that all other arguments to the macro are removed, but it does look pretty confusing and inconsistent when seen next to other macros such as `TFW_SKIP_COOKIE_NAME(chunk, end, n)`. Just an opinion.

Reply:
@krizhanovsky what do you think?

Reply:
I remove this macro at all